### PR TITLE
Updated getting started links

### PR DESCRIPTION
**Pull request by samie**
- Link to start.vaadin.com presets
- Added back Spring Initializr as an option

### CODE REVIEW

**Comment by samie** [comment]
We discussed the hasty removal of Spring Initializr links [in previous PR](https://github.com/vaadin/docs/pull/523#issuecomment-854863560), and as our Spring support is now [fixed on right level at Spring Initializr](https://github.com/spring-io/start.spring.io/commit/f4656e99a757b76a0e81d4a110eaf2a3f68042c7), it should be ok.

File: articles/flow/integrations/spring/tutorial-spring-basic.asciidoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ https://spring.io/projects/spring-boot[Spring Boot] speeds up the development pr
 [NOTE]
 See <<tutorial-spring-basic-mvc#,Using Vaadin with Spring MVC>> to learn how to use Vaadin in more traditional https://docs.spring.io/spring/docs/current/spring-framework-reference/web.html[Spring MVC] web application, without Spring Boot.
 
-The easiest way to create an application with Spring Boot and Vaadin is to start with a template application created by https://start.vaadin.com, but you can also add required dependencies manually to your project.
+The easiest way to create an application with Spring Boot and Vaadin is to start with a template application created by https://start.vaadin.com/?preset=lts[start.vaadin.com], but you can also configure Vaadin project at https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=11&groupId=org.vaadin&artifactId=example-app&name=example-app&description=Vaadin%20project%20for%20Spring%20Boot&packageName=org.vaadin.example-app&dependencies=vaadin[Spring Initializr] or just add the required dependencies manually to your project.
```
Comment (jouni) [inline articles/flow/integrations/spring/tutorial-spring-basic.asciidoc:17]:
```suggestion
https://start.vaadin.com/?preset=lts[Vaadin Start] is the easiest way to create an application with Spring Boot and Vaadin.
You can also configure and download a Vaadin project using
pass:[<!-- vale Vale.Spelling = NO -->]
https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=11&groupId=org.vaadin&artifactId=example-app&name=example-app&description=Vaadin%20project%20for%20Spring%20Boot&packageName=org.vaadin.example-app&dependencies=vaadin[Spring Initializr]
pass:[<!-- vale Vale.Spelling = YES -->]
or add the required dependencies manually to your project.
```

File: articles/flow/integrations/spring/tutorial-spring-basic.asciidoc [hunk 2]
```diff
@@ -14,11 +14,13 @@ https://spring.io/projects/spring-boot[Spring Boot] speeds up the development pr
 [NOTE]
 See <<tutorial-spring-basic-mvc#,Using Vaadin with Spring MVC>> to learn how to use Vaadin in more traditional https://docs.spring.io/spring/docs/current/spring-framework-reference/web.html[Spring MVC] web application, without Spring Boot.
 
-The easiest way to create an application with Spring Boot and Vaadin is to start with a template application created by https://start.vaadin.com, but you can also add required dependencies manually to your project.
+https://start.vaadin.com/?preset=lts[Vaadin Start] is the easiest way to create an application with Spring Boot and Vaadin.
+You can also configure and download a Vaadin project using
+https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=11&groupId=org.vaadin&artifactId=example-app&name=example-app&description=Vaadin%20project%20for%20Spring%20Boot&packageName=org.vaadin.example-app&dependencies=vaadin[Spring Initializr] or add the required dependencies manually to your project.
```
Comment (magi42) [inline articles/flow/integrations/spring/tutorial-spring-basic.asciidoc:19]:
```suggestion
You can also configure and download a Vaadin project using https://start.spring.io/#!type=maven-project&language=java&packaging=jar&jvmVersion=11&groupId=org.vaadin&artifactId=example-app&name=example-app&description=Vaadin%20project%20for%20Spring%20Boot&packageName=org.vaadin.example-app&dependencies=vaadin[Spring Initializr] or add the required dependencies manually to your project.
```

File: articles/flow/integrations/spring/tutorial-spring-basic.asciidoc [hunk 3]
```diff
@@ -14,11 +14,13 @@ https://spring.io/projects/spring-boot[Spring Boot] speeds up the development pr
...
 
 == Adding Dependencies
 
-Like many other tech stacks on Spring Boot, Vaadin provides a starter dependency that provide all essential modules and autoconfiguration. Only the `vaadin-spring-boot-starter` dependency is needed, but it is suggested to also declare the vaadin-bom if you need additional Vaadin dependencies. For production builds, it is also suggested to add the vaadin-maven-plugin, that generates the optimized JavaScript packages.
+Like many other tech stacks on Spring Boot, Vaadin provides a starter dependency that provide all essential modules and autoconfiguration. Only the `vaadin-spring-boot-starter` dependency is needed, but it is suggested to also declare the `vaadin-bom` if you need additional Vaadin dependencies. For production builds, it is also suggested to add the `vaadin-maven-plugin`, that generates the optimized JavaScript packages.
```
Comment (magi42) [inline articles/flow/integrations/spring/tutorial-spring-basic.asciidoc:23]:
```suggestion
Like many other tech stacks on Spring Boot, Vaadin provides a starter dependency that provides all essential modules and auto-configuration. Only the `vaadin-spring-boot-starter` dependency is needed, but it is suggested to also declare `vaadin-bom` if you need additional Vaadin dependencies. For production builds, it is also suggested to add `vaadin-maven-plugin`, which generates the optimized JavaScript packages.
```

**Review by magi42 — CHANGES_REQUESTED** [review]
Need to add auto-configuration to dictionary.

**Comment by jouni** [comment]
@magi42, your previous GH review is still blocking ("request changes"). I’ll merge it now regardless.

**Pull request merged**